Scripts/buildAndRun.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# build
+#  - add 'cci' tag
+#  - path: . (current directory)
+docker build -t cci .
+
+if [ $? -eq 0 ]
```

Comment:
That's a possible way, but on the other hand we can use `set -xxx` at the beginning of the file to ensure exit when necessary and catch/print what we should.

Reply:
I didn't wanted the script to exit immediately, only after a sound effect (build proess is about 3-4 minutes long, and i sometimes I didn't notice that it had finished). It was my first idea and it was useful several times, but I'm ready to use a better solution if any.

Reply:
It's ok for now. It's just a helping script. I did only a mention :)